MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/config.html

```diff
@@ -36,33 +36,35 @@
                 uniquePluginId: "8c95c4d2-e50c-4fb0-a4f3-6c06ff0f9a1a"
             };
 
-            $('.musicBrainzConfigPage').on('pageshow', function () {
-                Dashboard.showLoadingMsg();
-                ApiClient.getPluginConfiguration(MusicBrainzPluginConfig.uniquePluginId).then(function (config) {
-                    $('#server').val(config.Server).change();
-                    $('#rateLimit').val(config.RateLimit).change();
```

Comment:
I see no `change`-listeners here, so event emiting is not required, imo.

Reply:
Note that the listeners may be on the web client side.

Reply:
Looks like there is one. `emby-input` changes label style on `change`.

Reply:
I think I've been removing the change() functions everywhere. Should they be kept or does it not matter?

Reply:
We could add `valueset` handler for `emby-input`.
But if we don't want to do this right now, we should keep event emitting.

To "perfectly" emulate `change` event:
```js
elem.dispatchEvent(new Event('change', {
    bubbles: true,
    cancelable: false
}));
```